src/sidenav/SideNav.js

```diff
@@ -30,9 +30,10 @@ class SideNav extends React.Component {
         <Paper className="side-nav-paper">
           <MenuList className="side-nav-list">
             {this.renderMenuItem("Translations", ROUTES.TRANSLATIONS)}
-            {this.renderMenuItem("Settings", ROUTES.SETTINGS)}
+            {this.renderMenuItem("User contributions", ROUTES.CONTRIBUTIONS)}
```

Comment:
"User Contributions" (second word capitalized, to match other nav cases)

Reply:
Done